Tests/demistomock/demistomock.py

```diff
@@ -757,15 +757,15 @@ def setContext(contextPath, value):
 
 
 def demistoUrls():
-    """Retrieves the following URLs related to the incident:
+    """Retrieves the following URLs related to the incident.
+    You can use these URLs when you send notifications outside the system
+    over email or slack and would like to include a link to the incident.
         - evidenceBoard
         - investigation
         - relatedIncidents
         - server
         - warRoom
         - workPlan
-        You can use these URLs when you send notifications outside the system over email or slack
```

Comment:
its a bit weird the way it looks now
you can just dedent this section and it will show it was meant